src/components/TransactionsList/TransactionsList.js

```diff
@@ -44,26 +45,20 @@ class TransactionsList extends PureComponent<Props, State> {
     this.rootWrapper.addEventListener('scroll', this.handleScroll)
```

Comment:
why don't just `window`?

Reply:
Because scroll in our app set in [`#root` element](https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/styles/core.scss#L23), and scroll of window don't handle.